lib/recog/fingerprint.rb

```diff
@@ -248,13 +248,23 @@ def parse_description(xml)
 
   # @param xml [Nokogiri::XML::Element]
   # @return [void]
-  def parse_examples(xml)
+  def parse_examples(xml, filepath)
     elements = xml.xpath('example')
 
     elements.each do |elem|
       # convert nokogiri Attributes into a hash of name => value
       attrs = elem.attributes.values.reduce({}) { |a,e| a.merge(e.name => e.value) }
-      @tests << Test.new(elem.content, attrs)
+      if attrs["_filename"]
+        contents = ""
+        fn = File.join(filepath, attrs["_filename"])
+        File.open(fn, "rb") do |file|
+          contents = file.read.chomp
```

Comment:
There may be cases where the trailing new line is relevant for the match, maybe remove the chomp? For example, if its DER/BER encoded, the last \n may be relevant content (0x10)